pkg/ccl/backupccl/schedule_pts_chaining.go

```diff
@@ -213,36 +214,33 @@ func manageIncrementalBackupPTSChaining(
 	return err
 }
 
-func getSpansProtectedByBackup(
+func getTargetProtectedByBackup(
 	ctx context.Context, backupDetails jobspb.BackupDetails, txn *kv.Txn, exec *sql.ExecutorConfig,
-) ([]roachpb.Span, error) {
+) (*ptpb.Target, roachpb.Spans, error) {
```

Comment:
[nit] Name these return values.

Reply:
I'm actually in a camp of not using this syntactic sugar.  But I don't have a strong opinion on that.

Reply:
done.